copter/source/docs/rtl-mode.rst

```diff
@@ -12,12 +12,17 @@ to use and customize RTL mode.
 Overview
 ========
 
-When RTL mode is selected, the copter will return to the home location.
-The copter will first rise to :ref:`RTL_ALT <RTL_ALT>` before returning home or maintain the current altitude if the current altitude is higher than :ref:`RTL_ALT <RTL_ALT>`.  The default value for :ref:`RTL_ALT <RTL_ALT>` is 15m.
+When RTL mode is selected, the copter will return to the home location, or if rally points have been setup, the closet rally point.
+
+The copter will first rise a minimum of :ref:`RTL_CLIMB_MIN<RTL_CLIMB_MIN>` or to  :ref:`RTL_ALT <RTL_ALT>`, whichever is higher, before returning home.  The default value for :ref:`RTL_ALT <RTL_ALT>` is 15m. Under no circumstances will this altitude be below 2m.
+
+The altitude reference frame is set by either the rally point, if proceeding to one of those, or by :ref:`RTL_ALT_TYPE<RTL_ALT_TYPE>` to select relative to HOME, or Terrain. If :ref:`WPNAV_RFND_USE<WPNAV_RFND_USE>` = 1 and Terrain is selected, then rangefinder will be used.
 
 .. image:: ../images/RTL.jpg
     :target: ../_images/RTL.jpg
 
+If RTL is entered close to its return point, the altitude Copter climbs to may be limited tp avoid unneeded climb and descents. The :ref:`RTL_CONE_SLOPE<RTL_CONE_SLOPE>` parameter determines the slope of an invetred cone centered on the return point. This reduces the above return altitude according to: distance from return point * :ref:`RTL_CONE_SLOPE<RTL_CONE_SLOPE>`. So if the mode is entered 10m from the return point, using the default slope of "3", then the altitude rise would be limited to 30m before returning. It may be less depending on the other parameters, but no higher. If the slope were set to "0.5", then the initial climb would be no higher than to 5m altitude before proceeding to the return point. A value of "0" disables this limit. "0.5" is the minimum slope. Again, 2m is the minimum return altitude.
```

Comment:
"the atltitude **that the** Copter climbs to"

Reply:
thanks...ran through Grammarly and fixed it...
fyi, 'Copter' is used interchangeably throughout the wiki with 'the copter'....